app/controllers/samples_controller.rb

```diff
@@ -800,19 +800,39 @@ def samples_going_public
 
   def report_v2
     pipeline_run = select_pipeline_run(@sample, params[:pipeline_version])
-    if pipeline_run
-      background_id = get_background_id(@sample, params[:background])
-      render json: PipelineReportService.call(pipeline_run.id, background_id)
-    else
-      render json: {
-        metadata: {
-          pipelineRunStatus: "WAITING",
-          jobStatus: "Waiting to Start or Receive Files",
-          errorMessage: nil,
-          knownUserError: nil,
-        },
-      }
-    end
+    background_id = get_background_id(@sample, params[:background])
+
+    skip_cache = params[:skip_cache] || false
+    MetricUtil.log_analytics_event("samples.cache.requested", current_user) unless skip_cache
```

Comment:
For the `log_analytics_event` calls, could you changes the names to the other format? https://czi.quip.com/67RCAIiHN0Qc/IDseq-product-analytics-How-to-log

Reply:
Changed!